client/src/components/ProductText.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components/macro";
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 1rem;
```

Comment:
Sometimes you use rem and sometimes px as unit for margin and padding. If possible I would choose one for more consistency.

Reply:
Thanks! You're right! I changed my mind about using px/rem a few times. Will spend some time this week refactoring the code to make it more consistent!